docs/plugins/storage.md

```diff
@@ -0,0 +1,76 @@
+# Storage plugin
+
+Occasionally, you may want to persist data inside your App. By default, the browser's localStorage is used for that purpose. However, if localStorage is not supported by the platform, it automatically falls back to *Cookies*.
+
+Please note that the Cookie size is limited to 4096 Bytes.
+
+## Registering the plugin
+
+The Storage plugin is part of the core Blits package, but it's an optional plugin that needs to be registered in the App's `index.js`,
+as demonstrated in the example below.
+
+Make sure to place the `Blits.Plugin()` method _before_ calling the `Blits.Launch()` method
+
+```js
+// index.js
+
+import Blits from '@lightningjs/blits'
+// import the storage plugin
+import { storage } from '@lightningjs/blits/plugins'
+
+import App from './App.js'
+
+// Use the Blits Storage plugin
+Blits.Plugin(storage)
+
+
+Blits.Launch(App, 'app', {
+  // launch settings
+})
+```
+
+Within the application we can call the storage plugin methods as below
+
+```
+this.$storage.get(key, value)
+```
+
+## Available methods
+
+### set
+
+Saves a key-value combination in storage.
+
+```js
+this.$storage.set(key, value)
+```
+
+The key is expected to be a `String`. The value can be a `String`, `Object`, `Boolean` or `Array`.
```

Comment:
No numbers?

Reply:
@sandeep-vedam I think @jfboeve was referring to the _value_ also being a number and not so much the key?